### ISSUE TITLE
The remaining pages don't load after setting preloadPageCount

### ISSUE THREAD

**Issue by usmanazizgroupdocs** [groupdocs-viewer/GroupDocs.Viewer-for-Java-Spring#46]
Attached is the customized Spring application based on [this commit](https://github.com/groupdocs-viewer/GroupDocs.Viewer-for-Java-Spring/pull/41/files) that loads the documents from the URL. When the `preloadPageCount `property is set, the rest of the pages are not loaded and it keeps saying `Loading... please wait`. For example, if we have a 10 page PDF document and preloadPageCount is set to 2, the pages 3-10 are not displayed. Would you please check. Regards 

Attachment: https://drive.google.com/open?id=1a-H3e3fKcVKtDWZkOL9_mTswyw43ZJ5X

**Comment by koleslena** [comment]
Hi @usmanazizgroupdocs , we are investigating the problem and will return to you when we have any updates.

**Comment by koleslena** [comment]
Hi @usmanazizgroupdocs , the cause of the problem was a broken scrolling, it works if we use next page button, but with scrolling it doesnt work. I made the fix as a commit in this custom branch, use it to fix your problem or make following steps:
1.  Was increased version of viewer angular to newest 0.4.21
2. Added gdScrollable directive to gd-document components tag in custom-viewer.component.html
